cmd/cri-containerd/options/options.go

```diff
@@ -63,12 +64,24 @@ type CniConfig struct {
 	NetworkPluginConfDir string `toml:"conf_dir" json:"confDir,omitempty"`
 }
 
+// Mirror contains the config related to the registry mirror
+type Mirror struct {
+	Endpoint string `toml:"endpoint" json:"mirror,omitempty"`
```

Comment:
Why one endpoint, one mirror?

Reply:
A Mirror will have an endpoint and any other associated config

Reply:
I mean for toml and json, they are different.